project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject midje "1.9.0-alpha8"
+(defproject midje "1.9.0-alpha9"
```

Comment:
Figure this is worth releasing because https://github.com/marick/Midje/pull/390, released in `alpha8`, makes it more probable that users will hit the buggy logic in `merge-metaconstants`